src/com/itacademy/eshop/product/TV.java

```diff
@@ -0,0 +1,12 @@
+package com.itacademy.eshop.product;
+
+import com.itacademy.eshop.interfaces.Discountable;
+import com.itacademy.eshop.interfaces.Returnable;
+import com.itacademy.eshop.product.types.Category;
+
+public abstract class TV extends Electronics implements Discountable, Returnable {
```

Comment:
TV should no longer be abstract, in this case you should make it a regular class and implement the unimplemented methods